packages/repo-tools/src/commands/util.ts

```diff
@@ -24,14 +24,33 @@ import pLimit from 'p-limit';
 // just per-runner.
 const limiter = pLimit(os.cpus().length);
 
+/**
+ * Quotes an argument if it contains spaces or other characters that need escaping
+ */
+function quoteArgument(arg: string): string {
+  if (!arg) return arg;
+
+  // If the argument contains spaces, wrap it in double quotes
+  if (arg.includes(' ')) {
+    // Escape any existing double quotes in the argument
+    const escapedArg = arg.replace(/"/g, '\\"');
+    return `"${escapedArg}"`;
+  }
+
+  return arg;
+}
+
 export function createBinRunner(cwd: string, path: string) {
   return async (...command: string[]) =>
     limiter(
       () =>
         new Promise<string>((resolve, reject) => {
+          // Quote arguments that contain spaces
+          const quotedArgs = [path, ...command].map(quoteArgument);
+
           execFile(
             'node',
-            [path, ...command],
+            quotedArgs,
             {
               cwd,
               shell: true,
```

Comment:
Not sure why we're using shell here, might be for windows? But I think removing it might fix the issue too.